interview-questions.md

```diff
@@ -7,62 +7,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: A conditional statement is essentially a statement that says a said value is true under certain conditions or criteria. This is often written as "if" "then" statements.
 
+  Researched answer: Conditional statements such as if, else if and else execute code if a set of conditions are met.
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: Git is essentially the program used to write various types of code on your local computer. Github is the online website that allows people to push/pull and deploy their code on a website and allow other people to access that data and work on projects.
 
+  Researched answer: Git is a control system that developers use to help track different versions of code and collaborate with other developers. Github is a online website that allows users to put projects on a remote server. The difference is git is downloaded on your computer to work on code with different people and github is a platform that is used to deploy git projects.
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: Equality operators
```

Comment:
The equality operator is definitely one that will return a boolean value.  Can you think of any other operators that return a Boolean value?

Reply:
Relational and Logical